conf/timezones.ics

```diff
@@ -1919,11 +1937,11 @@ END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VTIMEZONE
 TZID:America/Caracas
-LAST-MODIFIED:20170601T083716Z
+LAST-MODIFIED:20170711T043529Z
 X-ZIMBRA-TZ-PRIMARY:TRUE
 X-ZIMBRA-TZ-MATCH-SCORE:100
-X-ZIMBRA-TZ-ALIAS:(GMT-04.30) Caracas
-X-ZIMBRA-TZ-ALIAS:(UTC-04.30) Caracas
+X-ZIMBRA-TZ-ALIAS:(GMT-04.00) Caracas
```

Comment:
not sure my understanding is correct or not, but this should be GMT-4.30
in the X-ZIMBRA-TZ-ALIAS property we are storing old offsets, so we will be able to track to the correct timezone entry when we find entry with old offset

Caracas was having GMT-4.30 offset before this update, so I guess we are breaking the compatibility here. I haven't checked but this can be true for other timezones also